blackjax/mcmc/integrators.py

```diff
@@ -23,13 +23,23 @@
 from blackjax.types import ArrayTree
 
 __all__ = [
+    "velocity_verlet_coefficients",
+    "mclachlan_coefficients",
+    "yoshida_coefficients",
+    "omelyan_coefficients",
```

Comment:
Remove? Are you planning to use it outside of integrators?

Reply:
Also, it seems like code duplication to have `isokinetic_velocity_verlet`, `velocity_verlet`, etc, when we could just use the coefficients with `generate_isokinetic_integrator` and `generate_euclidean_integrator`

Reply:
> Also, it seems like code duplication to have isokinetic_velocity_verlet, velocity_verlet, etc, when we could just use the coefficients with generate_isokinetic_integrator and generate_euclidean_integrator

The design choice is motivated by how we are envisioning the usage of integrators in the library. Currently the design is to have integrator being a static set.

> As another example, I often want to do benchmarks against different integrators, but want to just iterate over X_coefficients, and then use generate_isokinetic_integrator for MCLMC and generate_euclidean_integrator for HMC

You should iterate through the integrator objects instead

> For example, if I want to run scripts that try different integrators, and I want access to their number of gradient calls. I suppose the other option would be to have a dictionary like {"velocity_verlet": {"num_grads": ..., "name": ..., "order": ...} }. Is that what you'd recommend?

Yes given that these are static, you should put them in your script as static parameters. I dont yet see those are useful in the library outside of benchmarking.

Reply:
OK, that makes sense. I think my one point of disagreement is that nothing in the code "knows" that `velocity_verlet` and `isokinetic_velocity_verlet` are related. So I will have to have a dictionary of `{"euclidean": velocity_verlet, "isokinetic": isokinetic_velocity_verlet, ...}` when I want to compare each integrator on hmc vs mclmc, which I'm currently doing. This is a little painful, but not the end of the world

Reply:
Updated, as per request